scripts/run-in-ios-sim

```diff
@@ -20,7 +20,7 @@ trap cleanup EXIT
 
 echo Opening Simulator... >&2
 open -a Simulator --args -CurrentDeviceUDID "$uuid"
-sleep 5
+sleep 20
```

Comment:
Maybe just add a prompt and let the user tell you when it's ready?

Reply:
Or do that as a fallback maybe

Reply:
Probably can detect when it's open using applescript or whatever the modern equivalent is